fixity/fixity.py

```diff
@@ -208,7 +222,8 @@ def scanall(
     :param str report_url: The base URL to a server to which the report will be POSTed after the scan completes. If absent, the report will not be transmitted.
     :param report_auth: Authentication for the report_url. Tupel of (user, password) for HTTP auth.
     :param int throttle_time: Time to wait between scans.
-    :param bool force_local: If True, will will request the Storage Service to perform a local fixity check, instead of using the Space's fixity (if available).
+    :param bool force_local: If True, will request the Storage Service to perform a local fixity check, instead of using the Space's fixity (if available).
+    :param bool timestamp: If True, will display the local time stamp of the AIP fixity scan.
```

Comment:
Similar situation here to what I commented in the `parser.add_argument` change above.